modulefiles/module_base.gaea.lua

```diff
@@ -36,4 +38,13 @@ load(pathJoin("py-xarray", (os.getenv("py_xarray_ver") or "None")))
 setenv("WGRIB2","wgrib2")
 setenv("UTILROOT",(os.getenv("prod_util_ROOT") or "None"))
 
+--prepend_path("MODULEPATH", pathJoin("/gpfs/f5/ufs-ard/world-shared/global/glopara/data/git/prepobs/v" .. (os.getenv("prepobs_run_ver") or "None"), "modulefiles"))
+--load(pathJoin("prepobs", (os.getenv("prepobs_run_ver") or "None")))
+prepend_path("MODULEPATH", pathJoin("/gpfs/f5/ufs-ard/world-shared/global/glopara/data/git/prepobs/v1.1.0", "modulefiles"))
+load(pathJoin("prepobs", "1.1.0"))
```

Comment:
@aerorahul The new prepobs version Kate had me install is v1.1.0. The other systems are still using v1.0.2. Do you have a suggestion where to put prepobs_run_ver? Clearly not hardcoded like this...

Reply:
I see it Refs #2840.

Reply:
can I merge this PR?

Reply:
@aerorahul If you are okay with hard coded prepobs version for now until other systems update to v1.1.0?

Reply:
fine with me for now. @KateFriedman-NOAA any preference?

Reply:
Yeah that's fine. I will turn that hardcoded value back into the variable in the PR I will make soon to update `develop` to `obsproc/v1.2.0` and `prepobs/v1.1.0`.

Note to self in issue #2840.

Reply:
Thanks @KateFriedman-NOAA. @aerorahul I just updated my branch so should be ready to merge. Thanks.